docs/migration-guide.rst

```diff
@@ -23,12 +23,14 @@ Rasa 1.2 to Rasa 1.3
 
 General
 ~~~~~~~
-
+- Default parameters and architectures for both ``EmbeddingPolicy`` and
```

Comment:
what else does the user need to know about this change? e.g. how did model training time change, how did performance change, can I safely use the new component instead of the old one